portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,19 +57,26 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   }
 
   /**
-   * Converts an ArrayList<String> into a JSON string using manual String concatentation.
+   * Converts an ArrayList<String> into a JSON string.
    */
-  private String convertToJson(ArrayList<String> facts) {
+  private String convertToJson(ArrayList<String> comments) {
     String json = "{";
-    json += "\"cs\": ";
-    json += "\"" + facts.get(0) + "\"";
-    json += ", \"ling\": ";
-    json += "\"" + facts.get(1) + "\"";
-    json += ", \"psych\": ";
-    json += "\"" + facts.get(2) + "\"";
-    json += ", \"phil\": ";
-    json += "\"" + facts.get(3) + "\"";
-    json += "}";
+    json += "\"history\": [";
+    for (int i = 0; i < comments.size(); i++) {
```

Comment:
Rather than incrementing the index in this for loop, could this use for-each syntax?

Reply:
I wanted a special case for the first character, which you can't do in a for-each loop